falco.yaml

```diff
@@ -26,6 +26,8 @@ environment:
       - cmake
       - curl-dev
       - elfutils-dev
+      - falco-plugins
```

Comment:
Are these meant to be build-time dependencies? I assumed they'd be runtime dependencies.

Or do we copy everything we need from these?

Reply:
It makes sense to move them to runtime dependencies